src/components/EnvironmentBadge.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import styles from '../styles/styles';
+import CONFIG from '../CONFIG';
+import CONST from '../CONST';
+
+const EnvironmentBadge = () => {
+    // If we are on production, don't show any badge
+    if (CONFIG.EXPENSIFY.ENVIRONMENT === CONST.ENVIRONMENT.PRODUCTION) {
+        return null;
+    }
+
+    let badgeText = 'DEV';
+    let style = styles.badgeDanger;
```

Comment:
Let's give this a more descriptive name:

```suggestion
    let backgroundColorStyle = styles.badgeDanger;
```